.github/workflows/cicd.yml

```
@@ -46,3 +49,9 @@ jobs:
         uses: ./.github/workflows/cd.yml
         secrets:
             DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    publish:
+        name: Publish content
+        uses: ./.github/workflows/publish.yml
+        secrets:
+            DATABASE_URL: ${{ secrets.DATABASE_URL }}
```

actionlint (1.7.4)

secret "DATABASE_URL" is not defined in "./.github/workflows/publish.yml" reusable workflow. no secret is defined

(workflow-call)